doc/objectives.md

```diff
@@ -0,0 +1,18 @@
+# Study Objectives
+
+- **Is there a relationship between predator size (mass, length, fork-length) and prey size (mass, length, fork-length)?**
+  
+  There will be a trend of larger predators consuming larger prey. 
+  
+- **Does the same species of predator in different geographic locations exhibit the same feeding pattern? Are they eating the same sizes of prey in different locations?**
+  
+  Locations with more similar environmental factors will have predators exhibiting similar feeding preferences. 
+  
+- **Does geography (latitude, longitude, geographic location, specific habitat) influence predator and prey sizes (mass or length ratios)?**
+
+  Different locations will exhibit different patterns in predator and prey size. 
+
+- **Do environmental factors (depth, mean annual temperature, mean primary productivity, specific habitat) influence feeding intractions or predator and prey size relationships (mass, length, fork length)?**
+  
+  In areas in which environmental factors are similar, feeding interactions and predator/prey size relationships will show similar trends. 
+
```

Comment:
We could talk about what we think each of the variables will do?
For example: In locations with higher mean primary productivity, we'd expect trophic upsurge leading to larger predators and prey, or possibly a greater diversity of prey available to the predator.
-In contrast, the pelagic zone is nutrient poor, deeper, and generally supports a lower diversity and abundance of fish. This could lead to predator dependence on one kind of fish, as opposed to the competitive but diverse coastal shelves.
-Annual temperature: idk lol some fishies like it cold